packages/driver/src/codecs/context.ts

```diff
@@ -62,6 +67,19 @@ export class CodecContext {
     return NOOP;
   }
 
+  getContainerOverload<T extends ContainerNames>(
+    kind: T,
+  ): ContainerOverload<T> {
```

Comment:
I think we want the nullability of this return type in the signature, right?

```suggestion
  ): ContainerOverload<T> | null {
```

Otherwise, you're not required to check downstream.

Reply:
Oh, I just noticed that this mapped type is already `| undefined`, nevermind!